Dockerfile

```diff
@@ -29,24 +38,26 @@ RUN apt-get update && \
     apt-get -qy install --fix-missing --no-install-recommends \
         curl \
         && \
-    curl -sL https://deb.nodesource.com/setup_0.10 | bash -
+    curl -sL https://deb.nodesource.com/setup_iojs_2.x | bash -
```

Comment:
I guess we should also amend `INSTALL.rst` and `docs/getting-started/overlay.rst` WRT nodejs -> iojs change, in order to have consistent development environments (with and without Docker).